### PR TITLE
better hover effect and fixed alignment 

### PR DESCRIPTION
**Pull request by betelhemdemsis**
##issue number - #195 

## Description
1,  edited the hover effect of the logo and navbar  
2, edited the hover effect on footer 
3, fixed the alignment of the button and content on the feature.jsx 

## Type of PR

- [ ] Bug fix

## Screenshots / videos (if applicable)
Screenshot 2024-06-11 212139

## Checklist:
- [ ] I have performed a self-review of my code
- [ ] I have read and followed the Contribution Guidelines.
- [ ] I have tested the changes thoroughly before submitting this pull request.
- [ ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: src/components/Features.jsx [hunk 1]
```diff
@@ -32,118 +32,122 @@ const Container = styled.div`
...
-box-sizing: border-box;
+  text-align: center;
+  padding: 2rem;
+  background: rgba(255, 255, 255, 0.1);
+  box-shadow: 0 2px 6px rgba(65, 11, 16, 0.15);
+  backdrop-filter: blur(10px);
+  border-radius: 1rem;
+  width: calc(25% - 1rem);
+  max-width: 250px;
+  flex: 1 0 auto;
```
Comment (everly-gif) [inline src/components/Features.jsx:50]: redundant.  you have written `display:flex` again in line 52. Choose which to keep and remove.

File: src/components/Features.jsx [hunk 2]
```diff
@@ -32,118 +32,122 @@ const Container = styled.div`
...
-  
+  font-size: 1.5rem;
+  margin: 1rem 0;
+  color: var(--title-color);
 `;
 
 const Button = styled.div`
   display: inline-block;
-  background-color: rgb(48, 175, 91);
+  background-color: rgba(48, 175, 91, 0.85);
```
Comment (everly-gif) [inline src/components/Features.jsx:73]: why is this change necessary?

File: src/components/Features.jsx [hunk 3]
```diff
@@ -32,118 +32,122 @@ const Container = styled.div`
...
   border-radius: 0.5rem;
   font-weight: var(--font-semi-bold);
   transition: 0.3s;
   cursor: pointer;
   text-decoration: none;
+  margin-top: 1rem;
 
   &:hover {
-    background-color: rgb(43, 57, 74);
+    background-color: rgba(43, 57, 74, 0.85);
```
Comment (everly-gif) [inline src/components/Features.jsx:84]: why is this change necessary?

**Comment by everly-gif** [comment]
Everything else looks good

File: src/components/Features.jsx (hunk 3 above)
Comment (betelhemdemsis) [reply src/components/Features.jsx:84]: It's to make a color a lil bit transparent

**Comment by betelhemdemsis** [comment]
> Everything else looks good

@everly-gif   i've edited them

**Comment by everly-gif** [comment]
pls resolve conflicts

**Comment by betelhemdemsis** [comment]
> pls resolve conflicts

done

**Review by everly-gif — APPROVED** [review]
@Mansi168 , approving from my side do have a look and merge.

**Comment by Mansi168** [comment]
still there r conflicts

**Comment by betelhemdemsis** [comment]
resolved


> still there r conflicts

resolved

**Comment by Mansi168** [comment]
Thanks for the pr!
